torchgen/api/types/__init__.py

```diff
@@ -0,0 +1,77 @@
+from .types import (
```

Comment:
This is a big pain. Does `import *` not work? Then we only need to maintain accurate `__all__` on the submodules.

Reply:
Haven't tried `__all__` with `import *`. Let me try it out